README.md

```diff
@@ -309,8 +309,21 @@ Options to XSecureLock can be passed by environment variables:
     and fall back to XRandR 1.2. Not recommended.
 *   `XSECURELOCK_PAM_SERVICE`: pam service name. You should have a file with
     that name in `/etc/pam.d`.
-*   `XSECURELOCK_PARANOID_PASSWORD`: make `auth_x11` hide the password
-    length.
+*   `XSECURELOCK_PARANOID_PASSWORD`: (deprecated) makes `auth_x11` hide the
+    password length. Use `XSECURELOCK_PASSWORD_PROMPT` instead. (`1` translates
+    to `cursor`; `0` translates to asterisks)
+*   `XSECURELOCK_PASSWORD_PROMPT`: Choose password prompt mode:
+    *   `cursor`: shows a cursor that jumps around on each key press.
```

Comment:
Can you put example strings in the doc here?

Reply:
I'm not sure what you mean by this comment. Could you expand?

Reply:
Something like
`_________/______`
`********_`
etc.

Reply:
Added in 833cf0d2fe46d71014207395d23153d2688ea49c.

Reply:
(see https://github.com/configurator/xsecurelock/blob/prompts-pr/README.md#options)